gta-gallery/src/components/editUser/index.jsx

```diff
@@ -1,57 +1,87 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { useHistory } from 'react-router-dom';
-import { InputFields } from '../inputFields';
 import { Button } from '../button';
-import { editUser } from '../../services/userService';
+import { editUser, getCurrentUser } from '../../services/userService';
+import { Form, Input } from 'antd';
```

Comment:
Move it up. And why you are using here antd Input if you have created your custom one